sensor_array/analysis/parse_output.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
```

Comment:
maybe add acomment about what is supposed to happen? what the args are, what it does, what it returns?